event_dispatcher.rst

```diff
@@ -9,10 +9,10 @@ During the execution of a Symfony application, lots of event notifications are
 triggered. Your application can listen to these notifications and respond to
 them by executing any piece of code.
 
-Internal events provided by Symfony itself are defined in the
-:class:`Symfony\\Component\\HttpKernel\\KernelEvents` class. Third-party bundles
-and libraries also trigger lots of events and your own application can trigger
-:doc:`custom events </components/event_dispatcher>`.
+Symfony triggers several :doc:`events related to the kernel </reference/events>`
+while processing an HTTP Request, third-party bundles and libraries also trigger
+lots of events and your own application can trigger :doc:`custom events
+</components/event_dispatcher>`.
```

Comment:
I like this... but this sentence is too long. I think we need a period after `HTTP Request`. And then maybe the next sentence needs a little bit of work.